### DISCUSSION TITLE
Would like to have an option for mixed case buttons

### DISCUSSION THREAD

**Discussion by mark1502** [saadeghi/daisyui#2337] [Ideas / feature requests]
Sometimes I don't like that the btn class forces button text to be upper case, regardless of how it's written in the html. 

It would be great if there was another button class, something like 'btn-mixedcase' where the button text is presented as it is typed in the html, rather than being automatically converted to upper case.

**Comment by saadeghi** [comment]
Tailwind CSS class `normal-case` does that. It can reset the default uppercase style.

**Reply by mark1502** [reply]
That's good to know, thanks!
